<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Keeper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/window/image/main.png</normaloff>:/image/window/image/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QVectorTreeWidget" name="treeWidget">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>70</number>
       </attribute>
       <column>
        <property name="text">
         <string>Text</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>Описание</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Desc"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Passwd"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Sha1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Sha">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Md5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_md">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Save">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Копировать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNewjail"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionNewRoot"/>
    <addaction name="separator"/>
    <addaction name="actionNewChild"/>
    <addaction name="actionSaveChild"/>
    <addaction name="actionDeleteItem"/>
    <addaction name="separator"/>
    <addaction name="actionCopyToClipbrd"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="actionGenHardPass"/>
    <addaction name="actionGenEasyPass"/>
    <addaction name="separator"/>
    <addaction name="actionGetMD4"/>
    <addaction name="actionGetMD5"/>
    <addaction name="actionGetSha1"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionSendEmail"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNewjail">
   <property name="text">
    <string>Создать хранилище</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionNewRoot">
   <property name="text">
    <string>Создать раздел</string>
   </property>
  </action>
  <action name="actionNewChild">
   <property name="text">
    <string>Создать запись</string>
   </property>
  </action>
  <action name="actionSaveChild">
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionCopyToClipbrd">
   <property name="text">
    <string>Копировать в буфер обмена</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>О Qt5</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="actionGenHardPass">
   <property name="text">
    <string>Сгенерировать сложный пароль</string>
   </property>
  </action>
  <action name="actionGenEasyPass">
   <property name="text">
    <string>Сгенерировать лёгкий пароль</string>
   </property>
  </action>
  <action name="actionGetMD5">
   <property name="text">
    <string>Получить хеш MD5 </string>
   </property>
   <property name="toolTip">
    <string>Получить хеш MD5</string>
   </property>
  </action>
  <action name="actionGetSha1">
   <property name="text">
    <string>Получить хеш Sha1</string>
   </property>
   <property name="toolTip">
    <string>Получить хеш Sha1</string>
   </property>
  </action>
  <action name="actionGetMD4">
   <property name="text">
    <string>Получить хеш MD4</string>
   </property>
  </action>
  <action name="actionSendEmail">
   <property name="text">
    <string>Сообщить о проблеме...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVectorTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qvectortreewidget.h</header>
   <slots>
    <signal>canDestroy()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
